src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -225,7 +225,15 @@ private Schema buildLoadDatasetSchema() {
         "INSERT INTO `<project>.<dataset>.<stagingTable>`" +
             " (load_tag, load_time, source_name, target_path, state, file_id, checksum_crc32c, checksum_md5, error)" +
             " VALUES <load_history_array:{v|('<load_tag>', '<load_time>', '<v.sourcePath>', '<v.targetPath>'," +
-            " '<v.state>', '<v.fileId>', '<v.checksumCRC>', '<v.checksumMD5>', '<v.error>')}; separator=\",\">";
+            " '<v.state>', '<v.fileId>', '<v.checksumCRC>', '<v.checksumMD5>', \"<v.error>\")}; separator=\",\">";
```

Comment:
I think we need to sanitize the string,  not just changed the quoting. Otherwise, and error that has a double quote in it will fail. Better to place single quotes with another character.